src/Aura/Uri/Query.php

```diff
@@ -19,6 +19,22 @@
  */
 class Query extends \ArrayObject
 {
+
+    /**
+     *
+     * ArrayObject constructor used to setFlags
+     * @see https://github.com/auraphp/Aura.Uri/issues/28
+     *
+     * @param mixed $input
+     * @param integer $flags
```

Comment:
We normally use `int` instead of `integer` ?

Reply:
I will make the change. I did a search on this repo itself. Let it be consistent.